src/util/DeviceFingerprint.js

```diff
@@ -86,8 +82,11 @@ define(['q', 'okta'], function (Q, Okta) {
 
       // Attach listener
       window.addEventListener('message', onMessageReceivedFromOkta, false);
-      // Load devicefingerprint page inside the iframe
```

Comment:
why remove the comment? I wonder if shall add a bit more comments regarding why loading devicefingureprint inside a iframe?

Reply:
Added back and also as we spoke lets deal with refactoring this file in a later PR.